README.md

```diff
@@ -889,6 +889,38 @@ targets:
   - name: gem
 ```
 
+### AWS Lambda Layer
+
+The target will create a new public lambda layer in each available region with
+the extracted artifact from GitHub actions.
```

Comment:
Let's not mention GitHub actions specifically, it's just one of potential artifact providers.